src/website/views/components/data-table/examples/portal/_complex.pug

```diff
@@ -11,11 +11,7 @@ h3 Complex
             :data="table",
           )
             template(#statusdesc='payload')
-              i(:class="`chi-icon icon-${payload.due.icon} -icon--${payload.due.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.due.status }}
-              br
-              i(:class="`chi-icon icon-${payload.overdue.icon} -icon--${payload.overdue.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.overdue.status }}
+              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.statusInfo }}
```

Comment:
To render column description popover as a simple string, there is no need of having a custom template for it.
```
{
  head: {
    [code: string]: {
      label: string;
      sortable?: boolean;
      sortBy?: string;
      sortDataType?: 'string' | 'number' | 'date' | 'boolean';
      align?: 'left' | 'center' | 'right';
      allowOverflow?: 'visible' | 'hidden';
      key?: boolean;
      bold?: boolean;
      description?: string | DataTableColumnDescription; // description can be just a string. No need for custom payload + template.
      isPrintDisabled?: boolean;
    };
  };
  body: DataTableRow[];
}
```

Reply:
@janibolkvadze This particular example is not utilizing the types which you have mentioned. I have tried adding description but it renders on the table header.
<img width="682" alt="SS" src="https://user-images.githubusercontent.com/92325462/157152769-d722c06f-24b9-4aff-9815-b4d5efbba44e.png">

Reply:
<img width="783" alt="Screenshot 2022-03-08 at 11 17 44" src="https://user-images.githubusercontent.com/25136131/157216621-adb302df-9b84-4853-8a52-15797a7159a1.png">

Reply:
Thank you for clarifying.